File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] res = new int[nums.length];
        for (int i = 0; i < nums.length; i++) {
            int curProduct = 1;
            int j  = 0;
            while (j < nums.length) {
                if (j != i) {
                    curProduct = curProduct * nums[j];
                }
                j++;
            }
            res[i] = curProduct;
        }
        return res;
    }
}  

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
from _heapq import heapify
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        """
        [-1, 0, 1, 2, 3]
        [-3, -2, -1, 0, 1]
        """
        nums_negated = [-i for i in nums]
        print(nums_negated)
        heapq.heapify(nums_negated)

        for i in range(k):
            res = heapq.heappop(nums_negated)
        
        return -1 * res